src/server/routes/queueDelete.ts

```diff
@@ -109,7 +112,7 @@ export async function enqueueSavedItemIds(
       );
 
       if (sqsEntries.length === config.aws.sqs.batchSize) {
-        sqsSends.push(sqsSendBatch(sqsEntries));
+        sqsCommands.push(buildSqsCommand(sqsEntries));
```

Comment:
are we continuing to use `chunk`? 

5000 records of primary key comes to 20kb and batch sqs message is upto 256kb, so we removed chunk in the account-data-deleter.